test/e2e-tests.sh

```diff
@@ -35,4 +37,37 @@ header "Running Go e2e tests"
 go_test_e2e ./test || failed=1
 
 (( failed )) && fail_test
+
+go build github.com/tektoncd/cli/cmd/tkn
+
+./tkn pipeline list && fail_test "Failed to list pipelines"
+./tkn pipeline describe foo && fail_test "Failed to describe pipeline"
+#./tkn pipelinerun list && fail_test "Failed to list pipelinerun"
+./tkn pipelinerun describe foo && fail_test "Failed to describe pipelinerun"
+#./tkn pipelinerun logs foo && fail_test "Failed to show logs"
+./tkn task list && fail_test "Failed to list task"
+./tkn taskrun logs foo && fail_test "Failed to show logs"
+#./tkn taskrun list && fail_test "Failed to list taskrun"
+
+install_pipeline_crd
+
+# change default namespace to tektoncd
+kubectl create namespace tektoncd
+kubectl config set-context $(kubectl config current-context) --namespace=tektoncd
+
+# create pipeline, pipelinerun, task, and taskrun
+kubectl apply -f ./test/resources/output-pipelinerun.yaml
+kubectl apply -f ./test/resources/task-volume.yaml
+sleep 10
```

Comment:
instead of waiting for the arbitrary seconds, should it wait until respective resources state reaches to a certain state?
e.g. for ref. https://github.com/k8spatterns/examples/blob/master/advanced/Controller/config-watcher-controller.sh#L19
:thinking:

Reply:
probably we can also use `kubectl get  -w pr/<name> -o json`

Reply:
@hrishin Thanks for this suggestion. 
As it has to be in a loop so `kubectl get pr/<name> -o json` works